newsletter/admin.py

```diff
@@ -43,13 +44,22 @@
 
 from .settings import newsletter_settings
 
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
Just wondering why you chose to move the logger initialization below.

I know generally it is better to keep imports together. But then again, I found that simply keeping these two lines of logging code together as a single 'boilerplate' blob simplifies matters - and keeps the logger available for debugging import problems. What's your take on that?